android/app/src/main/java/com/example/airbnb/ui/search/ViewModel.kt

```diff
@@ -1,33 +1,41 @@
 package com.example.airbnb.ui.search
 
-import androidx.lifecycle.LiveData
 import androidx.lifecycle.ViewModel
-import androidx.lifecycle.asLiveData
 import androidx.lifecycle.viewModelScope
 import com.example.airbnb.data.repository.Repository
-import com.example.airbnb.network.dto.CloseTravelContents
+import com.example.airbnb.network.common.NetworkResponse
+import com.example.airbnb.network.dto.Region
 import dagger.hilt.android.lifecycle.HiltViewModel
-import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.channels.BufferOverflow
+import kotlinx.coroutines.flow.*
 import kotlinx.coroutines.launch
 import javax.inject.Inject
 
 @HiltViewModel
-class ViewModel @Inject constructor(private val repository: Repository): ViewModel() {
+class ViewModel @Inject constructor(private val repository: Repository) : ViewModel() {
     private val _heroImage: MutableStateFlow<String?> = MutableStateFlow("")
-    val heroImage: LiveData<String?> = _heroImage.asLiveData()
+    val heroImage: StateFlow<String?> = _heroImage
 
-    private val _closeTravel: MutableStateFlow<List<CloseTravelContents?>?> = MutableStateFlow(null)
-    val closeTravel: LiveData<List<CloseTravelContents?>?> = _closeTravel.asLiveData()
+    private val _closeTravel: MutableStateFlow<List<Region?>?> = MutableStateFlow(null)
+    val closeTravel: StateFlow<List<Region?>?> = _closeTravel
```

Comment:
그 자체로 nULLABLE 이면서 그 안에 있는 값도 nullable 인 상태는 불필요한 null 체크가 너무 많아집니다. 
```suggestion
    private val _closeTravel: MutableStateFlow<List<Region>> = MutableStateFlow(emptyList())
```

한번 요렇게 바꿔보시고, 코드들이 어떻게 바뀌는지 확인해보시면 좋겠습니다!